frontend/src/types.ts

```diff
@@ -1470,7 +1470,7 @@ export interface BillingV2PlanType {
     note: string | null
     unit: string | null
     product_key: ProductKeyUnion
-    current_plan?: any
+    current_plan?: boolean | null
```

Comment:
Based on `frontend/src/mocks/fixtures/_billing_v2.tsx` and confirming in the Network tab locally this is actually a boolean @xrdt. Just added a new commit and this should be good to go 😄